### ISSUE TITLE
After copy card, all lables are select

### ISSUE THREAD

**Issue by Michael2222** [wekan/wekan#2970]
Hello,

if I copy a card with label from one board to another board, all available lables are selected:

![Copy](https://user-images.githubusercontent.com/7198272/77438505-31955f80-6de6-11ea-961d-11680b4e289d.PNG)

to

![Paste](https://user-images.githubusercontent.com/7198272/77438530-3b1ec780-6de6-11ea-9876-e2f0f1112f37.PNG)

Regards
  Michael

**Comment by marc1006** [comment]
Hello @Michael2222, the reason for this behavior is that the label name is used to select the labels to be copied. Your labels all have an empty string as name, so all labels with the name "" are copied. The same behavior is seen when you move a card.

@xet7 Is this behavior intended?

**Comment by xet7** [comment]
@marc1006 

No. This is just a normal bug. Anyone if welcome to fix it. Most software has bugs.

**Comment by marc1006** [comment]
@xet7 Okay, then the problem is that the label name is used as an identifier.

**Comment by marc1006** [comment]
What should happen if some labels to be copied are not defined for the new board? Should they be auto-created?

**Comment by xet7** [comment]
@marc1006 

Yes.

**Comment by marc1006** [comment]
The issue https://github.com/wekan/wekan/issues/2705 is related to this issue.

**Comment by xet7** [comment]
Moved to here from #2705 

From @juliensl

> I've got it !
> It works at the version 2.48
> And it is broken at the version 2.49 ! 
> 
> Thank you Docker
